src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -1139,9 +1170,18 @@ private static byte[] getData(Profile p, int tagSignature) {
      * @throws IllegalArgumentException if a content of the {@code tagData}
```

Comment:
This is an existing issue, if it's an issue: should the text say *“**the** content”* instead of *“**a** content”*? @prrace

Reply:
Yes.

Reply:
Updated.